Update message delete event success to match with edit event

ZhiWei1010 opened the pull request:
This commit updates the delete event success message to match with the edit event success message for consistency.
- Corresponding test case in DeleteEventCommandSuccess has been updated as well

Fixes #101

ZhiWei1010 committed:
Update message delete event success to match with add event

ZhiWei1010 committed:
Update test case in DeleteEventCommandTest to match with new
delete message

guohuang88 commented on src/main/java/seedu/address/logic/commands/DeleteEventCommand.java:
```diff
@@ -103,9 +103,8 @@ public CommandResult execute(Model model) throws CommandException {
-                Messages.format(patientToDeleteEvent),
-                eventToDelete));
+        return new CommandResult(String.format(MESSAGE_DELETE_EVENT_SUCCESS, eventToDelete.name,
+                patientToDeleteEvent.getName(), targetEventIndex.getOneBased(), eventToDelete.date));
```
Should this be targetPatientIndex instead of targetEventIndex?

ZhiWei1010 replied:
Yes that is a good catch!

guohuang88 commented on src/test/java/seedu/address/logic/commands/DeleteEventCommandTest.java:
```diff
@@ -82,9 +81,8 @@ public void execute_deleteValidEventIndexFromValidPatient_success() throws Comma
-                Messages.format(editedPatient),
-                eventToDelete);
+        String expected = String.format(String.format(MESSAGE_DELETE_EVENT_SUCCESS, eventToDelete.name,
+                editedPatient.getName(), INDEX_FIRST_EVENT.getOneBased(), eventToDelete.date));
```
Should this be INDEX_FIRST_PATIENT instead of INDEX_FIRST_EVENT?

ZhiWei1010 replied:
Yes that is a good catch!

guohuang88 reviewed (commented):
I am not sure whether the ID in the message should be patient id or event id.

ZhiWei1010 committed:
Update MESSAGE_DELETE_EVENT_SUCCESS to be consistent with success
message of edit event command

Update the corresponding test case in DeleteEventCommandTest to be
consistent with the updated delete event success message.

guohuang88 reviewed (approved):
LGTM
